src/App.jsx

```diff
@@ -1,13 +1,162 @@
-
 import './App.css'
+import {useState} from "react";
+import CounterTile from "./components/CounterTile.jsx";
+import {countDown, countReset, countUp} from "./helpers/fruitCounter.js";
+import MainButton from "./components/MainButton.jsx";
+import {useForm} from "react-hook-form";
+import FormInputField from "./components/FormInputField.jsx";
 
 function App() {
+    const [fruitState, setFruitState] = useState({
+        strawberries: 0,
+        bananas: 0,
+        apples: 0,
+        kiwis: 0
+    });
+
+    const {register, handleSubmit} = useForm({
+        defaultValues: {
+            age: 0,
+            "moment": "daytime"
+        }
+    });
+
+    function handleFormSubmit(e) {
+        console.log(e, fruitState);
+    }
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+            <form onSubmit={handleSubmit(handleFormSubmit)}>
+                <fieldset className="formFruitCounter">
+                    <CounterTile
+                        title="🍓Aardbeien"
+                        counter={fruitState.strawberries}
+                        onCountDown={() => countDown(fruitState, setFruitState, "strawberries")}
```

Comment:
je geeft bijna 2x hetzelfde mee, je had dit kunnen oplossen door de fruitstate en de setFruitstate mee te geven aan de counter. 
En dan in het component verder gebruiken!